rancher/templates/NOTES.txt

```diff
@@ -1,5 +1,7 @@
 Rancher Server has been installed.
 
+NOTE: Rancher may take several minutes to fully initialize. Please standby while Certificates are being issued and Ingress comes up.
+
```

Comment:
is there anyway to monitor state? Like a log file to watch, etc?

Reply:
There is a --wait option, but it just waits for the pod to be in a Running state. It doesn't wait for the certs to be generated or the ingress to finish.